apps/camera/js/views/preview-gallery.js

```diff
@@ -125,6 +130,17 @@ return View.extend({
     this.els.previewMenu.classList.remove('visible');
   },
 
+  onFrameWheel: function(event) {
+    if (event.deltaMode !== event.DOM_DELTA_PAGE || event.deltaY) {
+      return;
+    }
+    if (event.deltaX > 0) {
+      this.fire('wheel:right');
+    } else if (event.deltaX < 0) {
+      this.fire('wheel:left');
```

Comment:
This function should emit a `swipe` event instead. See below in `swipeCallback`:

```
self.emit('swipe', direction);
```

Where `direction` is either 'left' or 'right'.